src/services/TestDataService.ts

```diff
@@ -875,27 +875,16 @@ export class TestDataService {
         configurationKey: string,
         configurationValue = {},
         salesChannelId = '',
-    ): Promise<SystemConfig> {
-
-        const systemConfigStruct = {
-            id: this.IdProvider.getIdPair().uuid,
-            configurationKey: configurationKey,
-            configurationValue: configurationValue,
-            salesChannelId: salesChannelId || null,
-        }
+    ): Promise<void> {
+        const data: Record<string, unknown> = {};
+        data[configurationKey] = configurationValue;
 
-        const response = await this.AdminApiClient.post('system-config?_response=detail', {
-            data: systemConfigStruct,
+        const response = await this.AdminApiClient.post(`_action/system-config?_response=detail&salesChanneldId=${salesChannelId}`, {
```

Comment:
Somehow we have to delete the systemConfig entry here. Should be built-in that into the test case?

Reply:
to be honest, I'd only allow setting it per sales channel basis and just unset all sales channel specific values at the end

Reply:
I've changed it to unset all sales channel specific values. The api is also changed, so that it only allow settings sales channel specific values